services/movie.py

```diff
@@ -1,11 +1,12 @@
+from django.db import transaction
 from django.db.models import QuerySet
-
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
Similarly, use `Optional[str]` for the `title` parameter to indicate that it can be `None`.